index.html

```diff
@@ -0,0 +1,723 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <script
+      src="https://kit.fontawesome.com/9875f697fa.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>NEWYORKTIMES</title>
+  </head>
+
+  <body>
+    <!--NAVIGATION MANU-->
+    <div class="topnav">
+      <div class="nav-left">
+        <a href="#"><i class="fas fa-bars fa-sm"></i></a>
+        <a href="#"><i class="fas fa-search fa-sm"></i></a>
+        <a href="#space&cosmos">SPACE & COSMOS</a>
+      </div>
+
+      <a href="#"
+        ><img
+          src="http://assets.stickpng.com/images/592fe3133919fe0ee3614d93.png"
+          alt="New York times"
+          class="newyork-logo"
+      /></a>
+
+      <div class="nav-right">
+        <button type="submit" class="btn info">SUBSCRIBE NOW</button>
+        <button type="submit" class="btn info">LOG IN</button>
+      </div>
+    </div>
```

Comment:
## Improve the Navigation menu 
- [ ] The link image logo is not loading. I'd suggest you check how you are referencing the images in the `img` tag. This is what we currently have. 👇 
- [ ] The social links should also be matching the original page and be centred on the page.

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://stephanie041996.github.io/Newyork/)|
|:--:|:--:|
|![header](https://user-images.githubusercontent.com/22345961/108398611-4367c880-722a-11eb-8db0-ef5fed2079a5.png)|![header-clone](https://user-images.githubusercontent.com/22345961/108398643-4cf13080-722a-11eb-88a3-10ad4112fc34.png)|